R/shrd_levels.R

```diff
@@ -123,10 +123,10 @@ AssignLevels <- function(segments, thr) {
     n = length(closest_indices)
     for (i in 1:n) {
       # Finding  segment i from `closest_indices` in `segments`
-      segments[which(segments[,1] == closest_indices[i]), 8] = level
+      segments[which((segments[,1] == closest_indices[i])), 8] = level
```

Comment:
Any reason for the double parentheses? AFAIK that is redundant no?

Reply:
Yep, typo.